server/routes/api.py

```diff
@@ -213,16 +215,19 @@ def post(author_id: str) -> Response:
                     {"message": "No visibility given for this post"},
                 )
         except KeyError:
+            print(0)
```

Comment:
maybe clean these prints since it will make things a bit confusing

Reply:
woops that's from when I was testing I'll remove it